examples/xgboost/train_configs/config_train_1_example.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data_path": "./dataset/HIGGS_UCI.csv",
+    "local_model_path": "model.json",
+    "global_model_path": "model_global.json",
+    "learning_rate": 0.1,
+    "objective": "binary:logistic",
+    "max_depth": 8,
+    "eval_metric": "auc",
+    "nthread": 16,
+    "data_index": {
+        "valid": {
+            "start": 0,
+            "end": 1000000
+        },
+        "site-1": {
+            "start": 1000000,
+            "end": 11000000,
+            "lr_scale": 1.0
+        }
+    }
+}
```

Comment:
suggest rename this file from 
`config_train_1_example.json`
to 
`config_train.json` to be consistent with your config_fed_client.json

Reply:
or just remove this file, what do you think?